dolphinscheduler-worker/src/main/java/org/apache/dolphinscheduler/server/worker/processor/TaskKillProcessor.java

```diff
@@ -124,6 +124,9 @@ public void process(Channel channel, Command command) {
     private void sendTaskKillResponseCommand(Channel channel, TaskExecutionContext taskExecutionContext) {
         TaskKillResponseCommand taskKillResponseCommand = new TaskKillResponseCommand();
         taskKillResponseCommand.setStatus(taskExecutionContext.getCurrentExecutionStatus().getCode());
+        if (taskExecutionContext.getAppIds() != null) {
+            taskKillResponseCommand.setAppIds(Arrays.asList(taskExecutionContext.getAppIds().split(TaskConstants.COMMA)));
+        }
         taskKillResponseCommand.setAppIds(Arrays.asList(taskExecutionContext.getAppIds().split(TaskConstants.COMMA)));
```

Comment:
```suggestion
        if (taskExecutionContext.getAppIds() != null) {
            taskKillResponseCommand.setAppIds(Arrays.asList(taskExecutionContext.getAppIds().split(TaskConstants.COMMA)));
        }
```

Reply:
OMG!! I'm really sorry. I made a low-level mistake. I've corrected it.